packages/tsx-editor/src/components/EditorContext.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import * as monaco from 'monaco-editor';
+import { ITextModel } from './Editor.types';
+
+interface IProviderProps {
+  code: string;
+}
+
+const defaultValues = {
+  model: monaco.editor.createModel(''),
+  id: '',
+  setModel: (newModel: ITextModel) => {
+    return;
+  },
+  setID: (newID: string) => {
+    return;
+  }
+};
+
+export const EditorContext = React.createContext(defaultValues);
+
+export class EditorProvider extends React.Component<IProviderProps> {
```

Comment:
What's the type of this component's state?

Reply:
Sorry, forgot to delete the file. It is no longer being used